src/actionConstants.js

```diff
@@ -85,3 +85,7 @@ export const DELETE_OUTBOX_MESSAGE = 'DELETE_OUTBOX_MESSAGE';
 
 export const DRAFT_ADD = 'DRAFT_ADD';
 export const DRAFT_REMOVE = 'DRAFT_REMOVE';
+
+export const WEBVIEW_CLEAR_MESSAGES_FROM = 'WEBVIEW_CLEAR_MESSAGES_FROM';
```

Comment:
We should not have any WEBVIEW-specific Redux events. Everything you are trying to do here, is doable in other ways.